pkgs/by-name/ko/koruri/package.nix

```diff
@@ -0,0 +1,30 @@
+{ lib
+, fetchFromGitHub
```

Comment:
Please format with the new formatter.

Reply:
Resolved by: bf1428225f7955c5d26f3a54f9c7c154b18ce24a